docs/catalog/triggers/kubernetes.rst

```diff
@@ -73,9 +73,16 @@ Additional triggers
 These triggers fire on very specific events:
 
 .. _on_kubernetes_warning_event:
+.. _on_kubernetes_warning_event_create:
+.. _on_kubernetes_warning_event_update:
+.. _on_kubernetes_warning_event_delete:
 
-* on_kubernetes_warning_event - when a Kubernetes event of level WARNING is created or modified
+* on_kubernetes_warning_event_create - when a Kubernetes event of level WARNING is created, modified, or deleted
```

Comment:
i think this should be without the _create

Reply:
Thanks, good catch